brave-http/src/main/java/com/github/kristofa/brave/http/HttpServerRequestAdapter.java

```diff
@@ -43,7 +44,9 @@ public String getSpanName() {
 
     @Override
     public Collection<KeyValueAnnotation> requestAnnotations() {
-        return Collections.emptyList();
+        KeyValueAnnotation uriAnnotation = KeyValueAnnotation.create(
+                "http.uri", serverRequest.getUri().toString());
```

Comment:
"http.url" is the current name, when query parameters are present (which I think this is..)